md/bdm-in-rest-api.md

```diff
@@ -0,0 +1,117 @@
+# How to manage BDM in custom REST APIs
+
+## Use case
+
+The example demonstrates best practices when developing a REST API Extension accessing [Business Data](define-and-deploy-the-bdm.md).  
+Special attention is focused on performance matters.
+
+Image below shows the BDM model used for the example use-case:
+
+![BDM model used](images/bdm_model_for_rest_api_01.png)
+
+Note that the relations from car to wheel1, wheel2, wheel3, wheel4 are **[lazy](define-and-deploy-the-bdm.md#lazy_eager_loading) relations**, meaning that the Wheels are **not retrieved** directly when retrieving
```

Comment:
This note is somewhat contradicted in the warning below the code example. For someone not familiar with the code it might be confusing.

I suggest we cut the sentence there:
 _Note that the relations from car to wheel1, wheel2, wheel3, wheel4 are **[lazy](define-and-deploy-the-bdm.md#lazy_eager_loading) relations**_

Then reword the warning after the code like : 
_Since wheel1, wheel2, wheel3, wheel4 are **[lazy](define-and-deploy-the-bdm.md#lazy_eager_loading) relations**, they are **not retrieved** directly when retrieving a Car. The retrieval of related Wheel objects is only done **when accessing the fields** (via getWheel1(), ...).
:::warning.......